.github/workflows/branch.yaml

```diff
@@ -0,0 +1,33 @@
+# uncomment if you need build a version of the app from the branch
```

Comment:
Is it supposed to be commented out?

Reply:
Yes, I just left the file for the next time we need it. I cannot leave it uncommented because the required final revert before merging the PR. You need to know what you are doing.